server/utils/nunjucksSetup.ts

```diff
@@ -99,6 +100,33 @@ export default function nunjucksSetup(app: express.Express, applicationInfo: App
   njkEnv.addFilter('activityLogDate', activityLogDate)
   njkEnv.addFilter('removeEmpty', removeEmpty)
   njkEnv.addFilter('toSlug', toSlug)
+
+  app.use((req, res, next) => {
+    /* eslint-disable */
+    njkEnv.addFilter('decorateFormAttributes', (obj: any, section: string, filterName: string) => {
```

Comment:
Not sure I understand this... but guessing it's just copied from the MOJ sortable table component?

Reply:
Its actually taken from the original prototype. Its a filter on a form attribute that sets the value in session and defaults the form to the value held in session. Was trying to make it easier. Ive tried to change it a bit to make it more readable. Can refactor later if needed